config/values.yml

```diff
@@ -138,3 +141,13 @@ log_cache_client:
 
 docker_registry:
   http_secret: "" #! A random piece of data used to sign state that may be stored with the client to protect against tampering. For production environments you should generate a random piece of data using a cryptographically secure random generator.
+
+kubernetes:
+  api:
+    url: ""
+kpack:
+  registry:
+    hostname: ""
+    repository: ""
+    username: ""
+    password: ""
```

Comment:
once ytt is able to configure individual libraries (that are pulled in by cf-for-k8s) we should be able to remove this values "mirroring" between cf-for-k8s and capi-k8s-release: https://github.com/k14s/ytt/issues/105

Reply:
Thanks for the heads-up that this is coming.